client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -298,166 +299,171 @@ export default function WPCheckout( {
 	const isDIFMInCart = hasDIFMProduct( responseCart );
 
 	return (
-		<CheckoutStepGroup
-			stepAreaHeader={
-				<CheckoutSummaryArea className={ isSummaryVisible ? 'is-visible' : '' }>
-					<CheckoutErrorBoundary
-						errorMessage={ translate( 'Sorry, there was an error loading this information.' ) }
-						onError={ onSummaryError }
-					>
-						<CheckoutSummaryTitleLink onClick={ () => setIsSummaryVisible( ! isSummaryVisible ) }>
-							<CheckoutSummaryTitle>
-								<CheckoutSummaryTitleIcon icon="info-outline" size={ 20 } />
-								{ translate( 'Purchase Details' ) }
-								<CheckoutSummaryTitleToggle icon="keyboard_arrow_down" />
-							</CheckoutSummaryTitle>
-							<CheckoutSummaryTitlePrice className="wp-checkout__total-price">
-								{ total.amount.displayValue }
-							</CheckoutSummaryTitlePrice>
-						</CheckoutSummaryTitleLink>
-						<CheckoutSummaryBody>
-							<WPCheckoutOrderSummary
-								siteId={ siteId }
-								onChangePlanLength={ changePlanLength }
-								nextDomainIsFree={ responseCart?.next_domain_is_free }
-							/>
-							{ ! isWcMobile && ! isDIFMInCart && <CheckoutSidebarPlanUpsell /> }
-							<SecondaryCartPromotions
-								responseCart={ responseCart }
-								addItemToCart={ addItemToCart }
-								isCartPendingUpdate={ isCartPendingUpdate }
-							/>
-							<CheckoutHelpLink />
-							<CheckoutNextSteps responseCart={ responseCart } />
-						</CheckoutSummaryBody>
-					</CheckoutErrorBoundary>
-				</CheckoutSummaryArea>
-			}
-		>
-			{ infoMessage }
-			<CheckoutStepBody
-				onError={ onReviewError }
-				className="wp-checkout__review-order-step"
-				stepId="review-order-step"
-				isStepActive={ false }
-				isStepComplete={ true }
-				titleContent={ <OrderReviewTitle /> }
-				completeStepContent={
-					<WPCheckoutOrderReview
-						forceRadioButtons={ forceRadioButtons }
-						removeProductFromCart={ removeProductFromCart }
-						couponFieldStateProps={ couponFieldStateProps }
-						onChangePlanLength={ changePlanLength }
-						siteUrl={ siteUrl }
-						createUserAndSiteBeforeTransaction={ createUserAndSiteBeforeTransaction }
-					/>
+		<>
+			<CheckoutOrderBanner />
```

Comment:
I only added `<CheckoutOrderBanner />` and wrapped everything in `<></>` so it has a parent element. The rest of the changes below are indentation due to the new `<>` parent element.

Reply:
I moved `<CheckoutOrderBanner />` up the stack so this is no longer accurate.